target/linux/mediatek/image/filogic.mk

```diff
@@ -322,6 +322,29 @@ define Device/cmcc_rax3000m
 endef
 TARGET_DEVICES += cmcc_rax3000m
 
+define Device/cmcc_rax3000m-nand-ubootmod
+  DEVICE_VENDOR := CMCC
+  DEVICE_MODEL := RAX3000M NAND version
```

Comment:
`NAND version` - this does not apply to the device model.

Reply:
Do you have any suggestions for device models?
Not sure about the rules here